methods_and_recursion/bottles_of_beer.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+
+ninetynine_beers = 99
+
+beer_on_the_wall = ninetynine_beers
+
+bot = 'bottles'
+
+while beer_on_the_wall > 2
```

Comment:
This hw was for a slide deck about methods, can you pull out some methods?  Perhaps `sing_a_verse(number)` and `pluralize(string, number)`?